src/Utilities.UnitTests/MockEngine.cs

```diff
@@ -36,8 +32,7 @@ sealed internal class MockEngine : IBuildEngine3
         private int _warnings = 0;
         private int _errors = 0;
         private int _commandLine = 0;
-        private string _log = "";
-        private string _upperLog = null;
+        private StringBuilder _log = new StringBuilder();
```

Comment:
It was there to be a basis for case-insensitive `Contains` comparisons. Which you can get by . . . using a case-insensitive comparer. And then the logs don't shout at you!